src/scenes/characters/bunny/bunny.gd

```diff
@@ -4,8 +4,9 @@ func _init():
     self.npc = "bunny";
 
 func _ready():
-    model = get_node("BunnyModel/bunnyMesh");
-    original_material = get_node("BunnyModel/bunnyMesh").get_surface_material(0).duplicate();
+    model = get_node("BunnyModel/1");
+    original_material = get_node("BunnyModel/1").get_surface_material(4).duplicate();
+    surface_material = 4;
```

Comment:
`original_material` should be derived from `model` and `surface_material` parameters. Feel free to refactor this in a separate PR.

Reply:
Moved to a separate issue, will refactor all character meshes there. Will also change to write over all available surfaces instead of a single surface.